recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -105,6 +105,7 @@ def setup_conda_rc(feedstock_root, recipe_root, config_file):
 
         call(["conda", "config", "--remove", "channels", "defaults"])
         for c in reversed(channels):
+            c = os.path.expandvars(c)
             call(["conda", "config", "--add", "channels", c])
```

Comment:
We’d be better off using the shell keyword argument and letting the shell do the variable expansion to avoid spilling tokens

Reply:
How would you do that?

Reply:
Oh the argument from `call()`! Hum... Would that work on non POSIX system?

Reply:
According to https://docs.python.org/3.8/library/os.path.html#os.path.expandvars, `os.path.expandvars()` will work on Linux, OSX and Windows using `$VAR_NAME`. I don't think  `$VAR_NAME` will work on Windows with `shell=True`.

Reply:
This indicates the expansion does not work on Windows (using POSIX like a variable): https://stackoverflow.com/questions/23304997/expanding-substituting-variables-in-a-popen-call#comment72190084_23305071

Reply:
Ouch. Idk how I have not hit this big before but ok.

Reply:
Note that I haven't verified the statement in the SO questions...